src/features/SignupView/Signup.Screen.js

```diff
@@ -72,12 +73,21 @@ const SignupScreen = () => {
     }
 
     signUpNewUser(email, password)
-      .then(userCredentials => {
+      .then(async userCredentials => {
         dispatch({ type: 'SET_AUTH', authenticated: true });
         dispatch({ type: 'SET_EMAIL', email });
         dispatch({ type: 'SET_ACCOUNT_HOLDER_NAME', accountHolderName });
         dispatch({ type: 'SET_NICKNAME', nickname });
         updateUserProfile();
+        try {
+          await addANewAnonymousDocument('users', {
+            name: accountHolderName,
+            email: email,
+          });
```

Comment:
I was wondering should we include `transactions` here as well so that we can refer to that field directly when create/join sessions? like default to an empty array. same for google/fb login

Also, we are not storing 'password' right? so basically we don't have verification if the user signs up through our app? But I guess that's fine lol.

Reply:
I also thought of including `transactions` here as an array. But later I thought, when we need to access the most recent item in the `transactions` array, we will not be able to do so, as I don't know how to access an index in an array in firestore. If you find any way to do this, we can definitely add this here. But if we keep it as a `transactions` subcollection like the `attendees`, then we can do it later when creating a new transition.
Yeah, we are not storing passwords now. Will it help for some use cases?

Reply:
Yeah I see your concerns now. I thought we are not accessing the most recent item anymore since we are not preserving the active session when user logs out. But we can definitely play around with this more later!

Storing password does not help in anyway, just asking because we had that in our schema lol.